Common/src/CConfig.cpp

```diff
@@ -3744,13 +3744,55 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
 /*--- Set default values for various fluid properties. ---*/
 
   const su2double Molecular_Weight_Default = 28.96;
+  const su2double Mu_Constant_Default = 1.716E-5;
+  const su2double Mu_Ref_Default = Mu_Constant_Default;
+  const su2double Mu_Temperature_Ref_Default = 273.15;
+  const su2double Mu_S_Default = 110.4;
+  const su2double Thermal_Conductivity_Constant_Default = 2.57E-2;
+  const su2double Prandtl_Lam_Default = 0.72;
+  const su2double Prandtl_Turb_Default = 0.9;
 
   if (Molecular_Weight == nullptr) {
     Molecular_Weight = new su2double[1];
     Molecular_Weight[0] = Molecular_Weight_Default;
     nMolecular_Weight = 1;
   }
 
+  if (Mu_Constant == nullptr) {
+    Mu_Constant = new su2double[1];
+    Mu_Constant[0] = Mu_Constant_Default;
+    nMu_Constant = 1;
+  }
```

Comment:
We have a rule for copy pasting...
```suggestion
  auto SetDefaultIfEmpty = [](su2double* array, unsigned short& size, const su2double& default) {
    if (array == nullptr) {
      array = new su2double[1];
      array[0] = default;
      size = 1;
    }
  };
  SetDefaultIfEmpty(Mu_Constant, nMu_Constant, Mu_Constant_Default);
```
Re-use for the other stuff